components/layout_2020/flexbox/layout.rs

```diff
@@ -1024,18 +1026,9 @@ impl<'a> FlexItem<'a> {
         );
 
         let pbm = box_.style().padding_border_margin(containing_block);
-        let content_box_size = box_
+        let (content_box_size, min_size, max_size, pbm) = box_
             .style()
-            .content_box_size_deprecated(containing_block, &pbm)
-            .map(|v| v.map(Au::from));
-        let max_size = box_
-            .style()
-            .content_max_box_size_deprecated(containing_block, &pbm)
-            .map(|v| v.map(Au::from));
-        let min_size = box_
-            .style()
-            .content_min_box_size_deprecated(containing_block, &pbm)
-            .map(|v| v.map(Au::from));
+            .content_box_sizes_and_padding_border_margin_deprecated(&indefinite_containing_block);
```

Comment:
I think this would suffice, without the new variable

```suggestion
            .content_box_sizes_and_padding_border_margin_deprecated(&containing_block.into());
```

Reply:
okay

Reply:
@Loirooriol pbm was arlready defined in scope and this new method returns pbm as part of the tuple. Should I clear the previous pbm variable?

Reply:
Yes, please clear it.